src/components/common/settings/SetText.tsx

```diff
@@ -31,15 +31,14 @@ const SetText: FC<Prop> = ({
       justifyContent="space-around"
       display="flex"
       alignItems="flex-end"
-      sx={{ margin: DEFAULT_MARGIN }}
     >
       <TextField
         data-cy={textDataCy}
         multiline={multiline}
         label={textFieldLabel}
         variant="outlined"
         onChange={handleChange}
-        sx={{ width: FULL_WIDTH, marginRight: DEFAULT_MARGIN }}
+        sx={{ width: FULL_WIDTH }}
```

Comment:
There is a prop `fullWidth` on the TextField component. [Ref](https://mui.com/material-ui/react-text-field/#full-width)